test/syncenginetestutils.h

```diff
@@ -634,8 +634,8 @@ class FakeChunkMoveReply : public QNetworkReply
         int size = 0;
         char payload = '\0';
 
-        do {
-            QString chunkName = QString::number(count).rightJustified(8, '0');
+        for (auto chunkName : sourceFolder->children.keys())
```

Comment:
:-1:   This makes the test void.  
The goal was to test that all the chunk files are there and they are named properly.

See the  `QCOMPARE(sourceFolder->children.count(), count); // There should not be holes or extra files` later, how can that fail with the current code?

So i'd just change the 8 with a 16.

Reply:
Yeah, that was on my TODO, to fix properly.

8 to 16 will not fix this problem, the problem is that `count` is no longer an auto incrementing number, it's the offset into the file now ...

Reply:
I'll need to implement a better way to check that all the `chunkId`'s to `chunkSize` match the range `0` to `item->_size`, with no holes.